### ISSUE TITLE
cannot downgrade pytorch to solve mismatch

### ISSUE THREAD

**Issue by tiangarin** [yjxiong/tsn-pytorch#109]
hello, i encounter mismatch error in load_state_dict, and i found someone suggest to downgrade to pytorch0.3.1. but my gpu is rtx2080ti, it can only support cuda10, which doesn't support pytorch 0.3.1.
so anyone can help me? I'm very worried now.

**Comment by tiangarin** [comment]
add:
it happen when i type:
`python main.py ucf101 RGB <ucf101_rgb_train_list> <ucf101_rgb_val_list> \
   --arch BNInception --num_segments 3 \
   --gd 20 --lr 0.001 --lr_steps 30 60 --epochs 80 \
   -b 128 -j 8 --dropout 0.8 \
   --snapshot_pref ucf101_bninception_ `

**Comment by mbilalshaikh** [comment]
yes, download the bn inception model and keep it locally. it solved the problem for me.
